apis/apps/v1alpha1/type.go

```diff
@@ -81,7 +80,6 @@ type ComponentTemplateSpec struct {
 	// This might be in conflict with other options that affect the file
 	// mode, like fsGroup, and the result can be other mode bits set.
 	//
-	// +kubebuilder:deprecatedversion:warning="This field has been deprecated since 0.9.0 and will be removed in 0.10.0"
 	// +optional
 	DefaultMode *int32 `json:"defaultMode,omitempty" protobuf:"varint,3,opt,name=defaultMode"`
```

Comment:
// The operator attempts to set default file permissions for scripts (0555) and configurations (0444).
// However, certain database engines may require different file permissions.
// You can specify the desired file permissions here.
//
// Must be specified as an octal value between 0000 and 0777 (inclusive),
// or as a decimal value between 0 and 511 (inclusive).
// YAML supports both octal and decimal values for file permissions.
//
// Please note that this setting only affects the permissions of the files themselves.
// Directories within the specified path are not impacted by this setting.
// It's important to be aware that this setting might conflict with other options
// that influence the file mode, such as fsGroup. 
// In such cases, the resulting file mode may have additional bits set.
// Refers to documents of k8s.ConfigMapVolumeSource.defaultMode for more information.

Reply:
updated